draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// Option tokens for Heart theme background colours.
+$ot-color-background-slate-10: rgba(var(--kz-var-color-slate-rgb-params), 0.1);
+$ot-color-background-white-10: rgba(var(--kz-var-color-white-rgb-params), 0.1);
+
+// Decision tokens for background colours based on presence of CSS variables.
+// If the `--kz-var-color-*` variable our `--ot-*` variables depend on is missing, the Zen tokens are used as a fallback.
+$dt-color-background-default: var(
```

Comment:
Super surprised that fallbacks can work within an `rgba` function like this. Very cool.

Reply:
Me too! I needed a codepen to convince myself it was real https://codepen.io/jasononeil/pen/vYyjzMz?editors=1100